.github/workflows/springwolf-plugins-bindings.yml

```diff
@@ -0,0 +1,56 @@
+name: springwolf-plugins-bindings
```

Comment:
Assuming that we only add bindings for existing plugins, I can imagine that we add it to the existing plugin pipeline